Device/ARM/ARMCM0plus/Source/system_ARMCM0plus.c

```diff
@@ -37,7 +37,7 @@
   Externals
  *----------------------------------------------------------------------------*/
 #if defined (__VTOR_PRESENT) && (__VTOR_PRESENT == 1U)
-  extern uint32_t __VECTOR_TABLE;
+extern const pFunc __VECTOR_TABLE[48];
```

Comment:
Should we move this one to the header file as well? Instead of duplicating the declaration by copy?

Reply:
That is OK with me. I can do that if everyone agrees.